File: PersonnelManagement/PersonnelManagement/Models/Decreeoperation.cs
using System;
using System.Collections.Generic;

namespace PersonnelManagement.Models
{
    public partial class Decreeoperation
    {
        public int Id { get; set; }
        public int Decree { get; set; }
        public int Subject { get; set; }
        public sbyte Created { get; set; }
        public sbyte Deleted { get; set; }
        public sbyte Changed { get; set; }
        public int Changedtype { get; set; }
        public DateTime? Dateactive { get; set; }
        public sbyte Datecustom { get; set; }
    }
}

File: PersonnelManagement/PersonnelManagement/Front/classes/decree/decree_operation.ts
export default class decree_operation {
    constructor() {
        this.id = 0;
        this.decree = 0;
        this.subject = 0; // У подразделений subject имеет знак минуса
        this.created = false;
        this.deleted = false;
        this.changed = false;
        this.changedtype = 0;
        
        this.dateactive = new Date();
        this.datecustom = false;
    }
    id: number;
    decree: number;
    subject: number; // У подразделений subject имеет знак минуса
    created: boolean;
    deleted: boolean;
    changed: boolean;
    changedtype: number;

    dateactive: Date;
    datecustom: boolean;
}